js_modules/dagit/packages/core/src/assets/AssetConfig.tsx

```diff
@@ -0,0 +1,35 @@
+import {gql} from '@apollo/client';
+
+import {
+  SidebarAssetQuery_assetNodeOrError_AssetNode_configField as ConfigField,
+  SidebarAssetQuery_assetNodeOrError_AssetNode_configField_configType_CompositeConfigType as AssetConfigSchema,
+} from '../asset-graph/types/SidebarAssetQuery';
+import {CONFIG_TYPE_SCHEMA_FRAGMENT} from '../typeexplorer/ConfigTypeSchema';
+
+export interface HasConfigField {
+  configField: ConfigField | null;
+}
+
+export function configSchemaForAssetNode(assetNode: HasConfigField): AssetConfigSchema | null {
+  if (assetNode.configField) {
+    const configSchema = assetNode.configField.configType as AssetConfigSchema;
+    return configSchema.fields.length > 1 ? configSchema : null;
+  } else {
+    return null;
```

Comment:
It seems like this logic should probably move to the backend rather than having to be filtered client-side? (eg can we return `null` instead of an empty object)

Reply:
Yes this needs some work, will have to do it in a follow-up though.